### PR TITLE
Enable recursive TV feature, using Vue.js templates

### PR DESCRIPTION
**Pull request by bobcao3**
Based on #17

### CODE REVIEW

File: src/script.ts [hunk 1]
```diff
@@ -164,15 +164,17 @@ function updateTheme(): void {
...
 function updateClock(): void {
     const time = new Date();
-    const clockTextElm = document.getElementById("clock-text");
+    const clockTextElms = document.getElementsByClassName("clock-text");
     let formattedTimeText = time.toLocaleString("en-US", { hour: "numeric", minute: "numeric", hour12: true });
     if (time.getSeconds() % 2) { // to make the blinking effect
         formattedTimeText = formattedTimeText.replace(":", " ");
     }
-    if (clockTextElm == null) {
+    if (clockTextElms == null) {
```
Comment (dkess) [inline src/script.ts]: This will never be null, so the if statement isn't necessary. Just loop through the objects in the collection.

**Review by dkess — COMMENTED** [review]
I would strongly recommend against using Vue here, since this isn't an interactive application. There should be easier ways to cut down on the code repetition, like by using raw Javascript calls.

**Comment by bobcao3** [comment]
> I would strongly recommend against using Vue here, since this isn't an interactive application. There should be easier ways to cut down on the code repetition, like by using raw Javascript calls.

Raw javascript call could be too hacky for this

File: src/script.ts (hunk 1 above)
Comment (JingkangZhang) [reply src/script.ts]: fixed

**Comment by bobcao3** [comment]
#17

**Pull request closed without merging**